native/packages/react-native-bpk-component-banner-alert/src/__snapshots__/BpkBannerAlert-test.ios.js.snap

```diff
@@ -540,18 +540,7 @@ exports[`iOS BpkBannerAlert should render correctly 1`] = `
     animateOnLeave={false}
     duration={200}
     show={true}
-    style={
```

Comment:
This change is actually highlighting where we were incorrectly applying `outerStyle` to both the banner alerts visual outer container as well as the animated height container.